app/src/components/market/common/additional_market_data/index.tsx

```diff
@@ -136,7 +136,7 @@ export const AdditionalMarketData: React.FC<Props> = props => {
           }
           isError={!verified}
         >
-          {verified ? <IconVerified size={'24'} /> : <IconAlert size={'24'} />}
+          {verified ? <IconVerified size={'24'} /> : <IconAlert size={'20'} />}
```

Comment:
Why change the size here?

Reply:
@KadenZipfel because with new svg size was not correct

Reply:
Well if the size is passed in as a prop then you can set it as whatever size right? So if it's normally supposed to be `size={'24'}` then should it be kept the same?

Reply:
@KadenZipfel you are right but svg size isn't same and dynamic viewBox size was also discarded

Reply:
@KadenZipfel what do you suggest should i change it to 24 ?

Reply:
I would set it as 24 since it was the previous size used, but I agree it does look a little off. See if you can find the size used for that icon in that spot on figma and go with whatever is used there